packages/frontend/src/redux/sharedThunks/refreshAccountOwner.js

```diff
@@ -26,7 +26,13 @@ export default createAsyncThunk(
                         break;
                     }
                 }
-                dispatch(makeAccountActive(nextAccountId));
+                if (nextAccountId) {
+                    dispatch(makeAccountActive(nextAccountId));
+                    //FIX: Automatic switching when account doesn't exist makes it impossible to switch to a zero balance account
```

Comment:
@marcinbodnar removing this automatic account switching when the account doesn't exist works fine until you refresh the page, because for some reason that causes the `availableAccounts` array in redux to be completely empty. Do you have any idea why that is? `wallet.accounts` still has the accounts listed, they're just not getting into the `availableAccounts` array. Should we maybe change the way we're loading `availableAccounts` to not depend on `refreshAccountOwner` and instead just pull them from `localStorage` without additional checking?

Reply:
Fixed it here: https://github.com/near/near-wallet/pull/2563/commits/bbf75775a3ccc2faa5c5256e8d1019aea6fb418f

Please review this solution